README.md

```diff
@@ -1,23 +1,81 @@
-# futures-timer
+<h1 align="center">futures-timer</h1>
+<div align="center">
+  <strong>
+    Timeouts for futures.
+  </strong>
+</div>
 
-[Documentation](https://docs.rs/futures-timer)
+<br />
 
-A library for working with timers, timeouts, and intervals with the `futures`
-crate.
+<div align="center">
+  <!-- Crates version -->
+  <a href="https://crates.io/crates/futures-timer">
+    <img src="https://img.shields.io/crates/v/futures-timer.svg?style=flat-square"
+    alt="Crates.io version" />
+  </a>
+  <!-- Downloads -->
+  <a href="https://crates.io/crates/futures-timer">
+    <img src="https://img.shields.io/crates/d/futures-timer.svg?style=flat-square"
+      alt="Download" />
+  </a>
+  <!-- docs.rs docs -->
+  <a href="https://docs.rs/futures-timer">
+    <img src="https://img.shields.io/badge/docs-latest-blue.svg?style=flat-square"
+      alt="docs.rs docs" />
+  </a>
+  <a href="https://discord.gg/JvZeVNe">
+    <img src="https://img.shields.io/discord/598880689856970762.svg?logo=discord&style=flat-square"
+      alt="chat" />
+  </a>
+</div>
 
-# License
+<div align="center">
+  <h3>
+    <a href="https://docs.rs/futures-timer">
+      API Docs
+    </a>
+    <span> | </span>
+    <a href="https://github.com/async-rs/futures-timer/releases">
+      Releases
+    </a>
+    <span> | </span>
+    <a href="https://async.rs/contribute">
+      Contributing
+    </a>
+  </h3>
+</div>
 
-This project is licensed under either of
+## Installation
+```sh
+$ cargo add futures-timer
```

Comment:
I think it is preferable to mention that `cargo-edit` is necessary.
Related: https://github.com/http-rs/surf/issues/102